makes/check-target.mk

```diff
@@ -54,7 +54,20 @@ endif
 
 # Check target
 ifeq (,$(filter $(TARGETS),$(TARGET_FAMILY)-$(TARGET_SUBFAMILY)))
```

Comment:
- please provide better indentation, maybe move defining `LF` and `SPACE` outside of `ifeq` (it might be even worth to move it to eg. `funcs.mk` to be potentially used in other makefiles). Maybe something like:
![image](https://user-images.githubusercontent.com/3939983/118618988-789d9b80-b7c4-11eb-9281-6a09e457859b.png)
- please fix the spelling error below

Reply:
Done